### ISSUE TITLE
nuget package

### ISSUE THREAD

**Issue by LAB02-Admin** [hexagon-oss/openhardwaremonitor#32]
Hey,

This is looking great and well maintained! Do you have plans to release it as a nuget package? Would be great to not have to manually add the dependencies to new projects!

Thanks either way for sharing 😃

**Comment by pgrawehr** [comment]
@LAB02-Admin I haven't actually published any official nugets so far... Also, honestly I have not tested/used it as a library, but only as a full application. 

There is an interface to the service in the `Iot.Device.Bindings` package from the dotnet team. This library offers functions for querying the sensors from a running OHM instance. Check out https://github.com/dotnet/iot/tree/main/src/devices/HardwareMonitor for instructions. 

What best suits your needs will depend on your use case. What are you exactly interested in?

**Comment by LAB02-Admin** [comment]
Hi @pgrawehr,

Thanks for your fast reply and info!

If you'd like to learn the process: using Visual Studio 2022, you can easily config your nuget package from the `Package` section in the project settings (in this case for `OpenHardwareMonitorLib`). Make sure you check `Produce a package file during build operations.`:

![image](https://user-images.githubusercontent.com/81011038/165517695-ca0cc6ef-603b-4bd5-b604-1d60ef6d706a.png)

Then after creating an account on [nuget.org](https://www.nuget.org) you can upload the .nuget file there:

![image](https://user-images.githubusercontent.com/81011038/165516867-765a2b98-4ae2-43a6-989c-127cecb77d88.png)

My use case is [HASS.Agent](https://github.com/LAB02-Research/HASS.Agent), a client application for the domotica system [Home Assistant](https://home-assistant.io/). One of its uses is to collect local machine sensors, and transmit them to the central server. I've been using the [LibreHardwareMonitorLib](https://github.com/LibreHardwareMonitor/LibreHardwareMonitor), but it doesn't get updated often. After looking around, I noticed your fork of openhardwaremonitor was recommended for being actively maintained.

So, basically, it's `OpenHardwareMonitorLib.dll` I'm after for using in another .NET 6 project, to get the same sensors values you show in your GUI, without an actual instance of OHM running.

**Comment by pgrawehr** [comment]
@LAB02-Admin Sorry for being unclear. Of course I know how to create packages. I have been using that a lot, but so far only with a private repository, never for the public nuget.org. I will need to look into that sometime anyway, but I hope it's not urgent, as I'm quite busy.

**Comment by PhyxionNL** [comment]
> I've been using the [LibreHardwareMonitorLib](https://github.com/LibreHardwareMonitor/LibreHardwareMonitor), but it doesn't get updated often.

Well, there's much more (hardware) activity there than here 🙂 If you need anything in particular implemented, feel free to send a PR 👀. Quality PRs are almost always accepted and merged 👍

**Comment by LAB02-Admin** [comment]
Alright, thanks @PhyxionNL!
